<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>630</width>
     <height>722</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>630</width>
     <height>722</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>630</width>
     <height>722</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Graphics/gambarAds.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="sentuhLayar">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>580</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>141</width>
     <height>41</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>141</width>
     <height>41</height>
    </size>
   </property>
   <property name="text">
    <string>Sentuh Layar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
